provider/pkg/provider/repository.go

```diff
@@ -17,13 +17,13 @@ package provider
 import (
 	"fmt"
 
-	containerregistry "github.com/pulumi/pulumi-azure-native/sdk/go/azure/containerregistry"
-	resources "github.com/pulumi/pulumi-azure-native/sdk/go/azure/resources"
+	"github.com/pulumi/pulumi-azure-native/sdk/go/azure/containerregistry"
+	"github.com/pulumi/pulumi-azure-native/sdk/go/azure/resources"
 	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
 )
 
-// The set of arguments for creating a RegistryGeoReplication component resource
-type RegistryGeoReplicationArgs struct {
+// The set of arguments for creating a RegistryArgs component resource
```

Comment:
We seem to have a confusion between repository and registry. It should be `Repository` everywhere because `Registry` has conflicts in `.NET`. 
```suggestion
// The set of arguments for creating a Repository component resource
```